doctrine/registration_form.rst

```diff
@@ -136,6 +145,15 @@ With some validation added, your class may look something like this::
             // You *may* need a real salt if you choose a different encoder.
             return null;
         }
+        
+        public function getRoles()
+        {
+            return $this->roles;
+        }
+
+        public function eraseCredentials()
+        {
+        }
 
         // other methods, including security methods like getRoles()
```

Comment:
If we make these changes, we'll need to reword or remove this comment too:

```
// other methods, including security methods like getRoles()
```

Thanks!

Reply:
Done 👍